infrastructure-playbooks/rolling_update.yml

```diff
@@ -821,6 +768,48 @@
     - import_role:
         name: ceph-client
 
+- name: complete upgrade
+  hosts:
+    - all
+  become: True
+  tasks:
+    - import_role:
+        name: ceph-defaults
+    - import_role:
+        name: ceph-facts
+
+    - name: container | disallow pre-nautilus OSDs and enable all new nautilus-only functionality
```

Comment:
wondering if we could avoid the duplicate task container/non-container with `{{ docker_exec_cmd_status|default('') }}`